src/components/Card.js

```diff
@@ -1,42 +1,80 @@
 export default class Card {
-  constructor(data, containerSelector, handleCardClick) {
-    this._data = data;
-    this._name = data.name;
-    this._link = data.link;
-    this._containerSelector = containerSelector;
+  constructor(
+    { cardData, myId, handleAPILikeClick, handleCardClick, handleDeleteClick },
+    cardSelector
+  ) {
+    this._cardData = cardData;
+    this._name = cardData.name;
+    this._link = cardData.link;
+    this._id = cardData._id;
     this._handleCardClick = handleCardClick;
+    this._handleDeleteClick = handleDeleteClick;
+    this._cardSelector = cardSelector;
+    this._handleAPILikeClick = handleAPILikeClick;
```

Comment:
This class should not know anything about the api. Please rename this method to `_handleLikeClick`